conference/react-js/src/containers/SignUpForm/index.jsx

```diff
@@ -34,107 +34,105 @@ export default class SignUp extends React.Component {
+          >
               password:
-            </label>
-            <input
-              type="password"
-              id="password"
-              name="password"
-              className="field sign-up__field"
-              required
-              minLength="6"
-              maxLength="30"
-            />
-            {/* {this.state.isValidCredentials ||
+          </label>
+          <input
+            type="password"
+            id="password"
+            name="password"
+            className="field sign-up__field"
+            required
+            minLength="6"
+            maxLength="30"
+          />
+          {/* {this.state.isValidCredentials ||
             (<span className="field-error">Please use at least one non-space
             character in your password</span>)} */}
-            <label htmlFor="confirm" className="form-label form-label_required">
+          <label htmlFor="confirm" className="form-label form-label_required">
               confirm password:
-            </label>
-            <input
-              type="password"
-              id="confirm"
-              name="confirm"
-              className="field sign-up__field"
-              required
-              minLength="6"
-              maxLength="30"
-            />
-            {this.state.password !== this.state.confirm ?
-              (<span className="field-error">Passwords do not match</span>)
-              : null}
-            {/* {this.state.isValidCredentials ||
+          </label>
+          <input
+            type="password"
+            id="confirm"
+            name="confirm"
+            className="field sign-up__field"
+            required
+            minLength="6"
+            maxLength="30"
+          />
+          {this.state.password !== this.state.confirm ?
+            (<span className="field-error">Passwords do not match</span>)
+            : null}
+          {/* {this.state.isValidCredentials ||
             ( <span className="field-error">Passwords do not match</span>)} */}
-            <input
-              type="submit"
-              className="sign-up__button btn"
-              value="submit"
-            />
-          </form>
-        </div>
-        <div className="pop-up-wrapper">
-          <div className="pop-up">
-            <h3 className="pop-up__title">
-              You&#39;ve successfully registered.
-            </h3>
-            <p className="pop-up__notification">
-              <span className="pop-up__notification-user" />, go to
-              <span className="pop-up__notification-user" />
-              to complete the sign-up process</p>
-            <button className="btn pop-up__button" >OK</button>
-          </div>
-        </div>
-      </section>
+          <input
+            type="submit"
+            className="sign-up__button btn"
+            value="submit"
+          />
+        </form>
+        {/* <div className="pop-up-wrapper"> */}
```

Comment:
why this code is commented?

Reply:
because, until the form submit functionality is realized

Reply:
okey